recipes/default.rb

```diff
@@ -75,7 +75,18 @@
     group node['ntp']['conf_group']
     mode '0644'
     source 'ntp.leapseconds'
+    source node['ntp']['leapfile_url']
     notifies :restart, "service[#{node['ntp']['service']}]"
+    not_if node['ntp']['leapfile_from_mirror']
```

Comment:
This check should wrap the resources instead of being done with a not_if. Using the not_if means these resources show up in every log file even if the user isn't using the feature.